### PR TITLE
Update `ru_ru.json`

### PR DESCRIPTION
**Pull request by mpustovoi**
Fixed typos, case and synced with `en_us.json`

### CODE REVIEW

File: src/main/resources/assets/farmersdelight/lang/ru_ru.json [hunk 1]
```diff
@@ -206,12 +206,12 @@
   "effect.farmersdelight.comfort": "Комфорт",
 
   "effect.farmersdelight.nourishment.0.desc": "Предотвращает потерю голода, за исключением случаев использования насыщения для лечения.",
-  "effect.farmersdelight.comfort.0.desc": "Делает вас невосприимчивым к Голоду, Медлительности и Слабости.",
+  "effect.farmersdelight.comfort.0.desc": "Делает вас невосприимчивым к голоду, медлительности и слабости.",
```
Comment (vectorwing) [inline src/main/resources/assets/farmersdelight/lang/ru_ru.json]: This description is outdated. Comfort has a different effect now:
"Provides natural regeneration, regardless of how hungry you are."

**Review by vectorwing — COMMENTED** [review]
@mpustovoi, I found a small discrepancy that you might want to look into. Otherwise, the translation is good to go.

Please let me know if you can change it this week, or if I can proceed with a merge anyway. 👍

File: src/main/resources/assets/farmersdelight/lang/ru_ru.json (hunk 1 above)
Comment (mpustovoi) [reply src/main/resources/assets/farmersdelight/lang/ru_ru.json]: Done.

**Comment by mpustovoi** [comment]
Just fixed it.

**Comment by vectorwing** [comment]
Thank you for the quick update! ^^

**Comment by mpustovoi** [comment]
I suppose this should also be copied to the 1.21 branch.
I can make another pull request, though.

**Comment by vectorwing** [comment]
Given I haven't introduced new langs to 1.21, I can just port all translations forward to it, which will be done after I go through them all. But if you'd like, feel free to open a PR there too!

**Pull request merged**